webview/src/experiments/components/table/Table.tsx

```diff
@@ -3,123 +3,12 @@ import { useSelector } from 'react-redux'
 import cx from 'classnames'
```

Comment:
I put each file in the `table` directory, but we could also put each file in one folder called `table`.